content/common/language-guide-overview.md

```diff
@@ -0,0 +1,533 @@
+title = "Language Guide Overview"
+template = "common_main"
+date = "2022-01-01T00:00:01Z"
+enable_shortcodes = true
+[extra]
+
+---
+
+The following code block outlines, in the context of the Spin framework, what languages have SDK support, and what SDK features are available in each of those languages:
```

Comment:
Suggest something like "This page" rather than "The following code block," and maybe rewording away from "in the context of the Spin framework".  How about something like "This page contains information about language support for Spin features."

Reply:
Sounds great @itowlson 
Updated in next push to this branch.
I like how the text matches the page's title (which will also eventually match the page's name); succinct and to the point!